### PR TITLE
feat(wallet): add disaster recovery config UI

### PR DESCRIPTION
**Pull request by olaszakos**
Adds new card on the Administration page if the user has SystemInfo privilege:
image

It'll show a Configure button, if the user has ChangeCanister privilege.

Opens the DR config dialog:
image

When DR is configured, it'll show the details:
image

It'll mark the DR user group on the User groups page, and will disable the delete button.
image

Todo:
- [x] add translation

### CODE REVIEW

File: apps/wallet/src/components/settings/DisasterRecoveryForm.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  get: () => modelValue.value.quorum,
+  set: value => {
+    if (value && typeof value !== 'number') {
+      value = parseInt(value, 10);
+    }
+
+    modelValue.value.quorum = value;
+    modelValue.value = {
+      ...modelValue.value,
+    };
```
Comment (keplervital) [inline apps/wallet/src/components/settings/DisasterRecoveryForm.vue:78]: nit: if there would be something deeply watching that value this change would avoid it to trigger the change twice.

```suggestion
    modelValue.value = {
      ...modelValue.value,
      quorum: value,
    };
```

**Review by keplervital — APPROVED** [review]
LGTM

File: apps/wallet/src/components/settings/DisasterRecoveryCard.vue [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          <VListItemSubtitle>
+            {{ systemInfo.disaster_recovery[0].user_group_name[0] || $t('terms.unknown') }}
+          </VListItemSubtitle>
+        </VListItem>
+        <VListItem class="px-0">
+          <VListItemTitle class="font-weight-bold"
+            >{{ $t(`terms.user_group_id`) }}:
+          </VListItemTitle>
+          <VListItemSubtitle>
+            {{ systemInfo.disaster_recovery[0].committee.user_group_id }}
```
Comment (keplervital) [inline apps/wallet/src/components/settings/DisasterRecoveryCard.vue:25]: nit: would be nice to have here the copy to clipboard btn

Comment (olaszakos) [reply apps/wallet/src/components/settings/DisasterRecoveryCard.vue:25]: added 👍

**Pull request merged**